performance-test/results/results.md

```diff
@@ -0,0 +1,291 @@
+# Performance Test Results
+
+Following the release of Data Prepper 1.3, performance tests were run for each of the new processors that were added, as well as the equivalent Logstash filters. In the performance test results discussed below, the test environments and configurations are identical, except where the same option is not available for all applications.
+
+## Environment Details
+
+### Data Prepper Environment
+
+![](../../docs/images/PerformanceTestEnvironment.png)
+
+### Logstash Environment
+
+![](../../docs/images/PerformanceTestEnvironmentLogstash.png)
+
+Comparing the performance of the latest release of Data Prepper 1.3 against Logstash 7.13.2, the test was configured to simulate 10 clients to send requests as frequently as possible. Each request was contained a batch of 200 logs. The test ran for 20 minutes, measuring the latency and throughput.
```

Comment:
Logstash 7.13.2 was released in June '21. There have been +20 versions since then. Logstash is on 8.1.2 as of a few weeks ago. Can we run these on a more recent version? That will be a better comparison for the community.

Reply:
There have been some issues getting the latest versions of Logstash running in our performance testing environment. I propose that we revisit this on the next round of performance testing